protocols/src/N2NP.nroff

```diff
@@ -164,8 +168,12 @@ a length of 4 bytes. The original hashed content and the size of those fields
 are not finally decided yet, so those are subject to change.
 In a packet, the protocol shall define a source identifier (FROM) and a
 destination identifier (TO). These fields are to be used by the neighboring
-nodes in order to relay the packet to its final destination.
-A routing algorithm could be used as well in order to accelerate this process.
+nodes in order to relay the packet to its final destination. Since a node can
+have multiple neighbors, and assuming the physical layer of the protocol
+(radio) can't handle a non-broadcast emission, a third ID is used; It
+describes the physical neighbor that will relay this packet. If the final
+destination node is within physical reach, this field is useless, and should
+be ignored.
```

Comment:
'ignoring' a field in protocol is simply skipping it at unpacking-time. The value of the field is not relevant, since it is not even read. But it has to be there, so i'll add this information

Reply:
let's assume the following, node a has a packet for node b and has the neighbors b and c

if the value doesn't matter it could, maybe even by accident, set it to the id of node c and node c would relay it to node b a second time (on top of node a having sent it already) wouldn't it?

Reply:
We'll have to be quite unlucky for that, but I see your point. The only solution is to set the field value to `0` then.

Reply:
which is what i would have gone for

as a bonus, that way a node can just take a look at the target and if it's not their id or 0 they can safely ignore it :D

Reply:
I think this comportment is not explicit enough in the specification; I added a section about it.

Reply:
The way i have it implemented in the simulator right now, i just set the nextNodeId (or however you wanna call it) to the next node. And if the next node is also the final node, so be it.